engine/android/app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">OpenBOR</string>
+<!--
+    <string name="app_name">Punch Man</string>
```

Comment:
Again, please allow me to chime in. This line might possibly not intentional. :)

Reply:
Everything was intended I have tested and it and its working.  The sign settings are there to use as a template (they wont be used on a debug build) and I have commented out app_name to avoid a conflict.